app/views/prompts/_prompt_controls.html.erb

```diff
@@ -43,7 +43,7 @@
           <li><%= link_to ts("Fulfill"), new_collection_work_path(collection, :claim_id => claim.id) %></li>
         <% end %>
         <% # The drop claim link should show up if the current user has claimed the prompt %>
-        <li><%= link_to ts("Drop Claim"), collection_claim_path(collection, claim), data: {confirm: ts('Do you really want to delete this claim?')}, :method => :delete %></li>
+        <li><%= link_to ts("Drop Claim"), collection_claim_path(collection, claim), data: {confirm: ts('Do you really want to drop this claim?')}, :method => :delete %></li>
```

Comment:
If you have the opportunity, could you make a few style changes to this line of code? (We like to sneak those in when we can.)

* `data: {confirm: ts('Do you really want to drop this claim?')}` should have some spaces inside the braces and use double quotes, i.e. `data: { confirm: ts("Do you really want to drop this claim?") }`
* `:method => :delete` can be written with newer syntax, i.e. `method: :delete`

If you don't have time, though, no worries! We can merge it as-is.

Reply:
not a problem at all! I see a few other spots in this file with the older stylings, so I can update those as well as long as I'm in here.

Reply:
Terrific, thank you!